device_doctor/lib/src/android_device.dart

```diff
@@ -164,6 +165,24 @@ class AndroidDeviceDiscovery implements DeviceDiscovery {
     }
     return healthCheckResult;
   }
+
+  @visibleForTesting
+  Future<HealthCheckResult> developerModeCheck({ProcessManager processManager}) async {
+    HealthCheckResult healthCheckResult;
+    try {
+      final String result = await eval(
+          'adb', <String>['shell', 'settings', 'get', 'global', 'development_settings_enabled'],
+          processManager: processManager);
+      if (result == '1') {
```

Comment:
Can you please add a comment describing this is the actual output command not the exit code and 1 means enabled?

Reply:
Added.